docs/docs/feature-library/flex-v2/29_keyboard_shortcuts_manager.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_label: keyboard-shortcuts-manager
+sidebar_position: 30
+title: keyboard-shortcuts-manager
```

Comment:
The `sidebar_label` needs to match the feature name `keyboard-shortcuts`.

Reply:
Understood.